app/src/main/java/ar/com/wolox/android/example/ui/home/news/NewsFragment.kt

```diff
@@ -10,6 +14,27 @@ class NewsFragment @Inject constructor() : WolmoFragment<FragmentNewsBinding, Ne
     override fun layout() = R.layout.fragment_news
 
     override fun init() {
+        with(binding) {
+            recyclerView.layoutManager = LinearLayoutManager(context)
+            val newsAdapter = NewsAdapter(newsList(resources))
+            recyclerView.adapter = newsAdapter
+            swipeRefresh.setColorSchemeResources(R.color.colorAccent)
```

Comment:
Can you implement this in the XML?

Reply:
Nope, swipeRefresh color setting is only available outside the XML